diagnostic_common_diagnostics/diagnostic_common_diagnostics/hd_monitor.py

```diff
@@ -82,8 +82,8 @@ def __init__(self):
         super().__init__(f'hd_monitor_{cleaned_hostname}')
 
         self._path = '~'
-        self._free_percent_low = 0.05
-        self._free_percent_crit = 0.01
+        self._free_percent_low = 5
+        self._free_percent_crit = 1
 
         self.add_on_set_parameters_callback(self.callback_config)
         self.declare_parameter('path', self._path)
```

Comment:
Could you please add explicit types here? Then the test would also work better

Reply:
you mean integer types for the parameters?

Reply:
What do you think? In practice an integer percentage should be enough, right? On the other hand it seems artificially restricting. But in your design you assumed integers, right?

Reply:
Yes you are right, I changed it now to explicit as integer.
Side-effect: If someone had used this parameter before with the fraction/ratio, this now raises an error. (which is good)